src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -1560,7 +1560,13 @@ void os::print_os_info(outputStream* st) {
 }
 
 void os::pd_print_cpu_info(outputStream* st, char* buf, size_t buflen) {
-  // Nothing to do for now.
+  char model[100];
+  size_t size = sizeof(model);
+  int mib[] = { CTL_HW, HW_MODEL };
```

Comment:
How portable is this across BSDs?

Reply:
It is portable included macOS and even DragonflyBSD.